AN_FinalDataAnalysisFunctions/PlotColAnalysisQuadrAN.m

```diff
@@ -0,0 +1,66 @@
+% plots the fractions of gene-positive cells, gene-positive colonies and number of cells as a function of colony size 
+% see also:  PlotColAnalysisFullChipAN
+% colonies is a cell array of colony data, separate for each of Nplot
+% quadrants/matfiles
+
+
+function [totalcells]=PlotColAnalysisQuadrAN(Nplot,colonies,M,thresh,nms2,param1,index1)
+
```

Comment:
this function is almost identical to PlotColAnalysisFullChipAN, I'm not even sure if we need separate function. just write 1 function that supports an arbitrary nplot and matfiles. For full chip it will only be 1 file, for quadrants it will be 4 but the code is the same.